test/karma-test-shim.js

```diff
@@ -66,6 +66,7 @@ System.config({
     '@angular/cdk/scrolling': 'node:@angular/cdk/bundles/cdk-scrolling.umd.js',
     '@angular/cdk/stepper': 'node:@angular/cdk/bundles/cdk-stepper.umd.js',
     '@angular/cdk/table': 'node:@angular/cdk/bundles/cdk-table.umd.js',
+    '@angular/cdk/tree': 'node:@angular/cdk/bundles/cdk-tree.umd.js',
```

Comment:
Why are we including the tree ? It is not used in our demos ?
> Then again neither is table ;-)

Reply:
It's required for testing, otherwise Karma will try looking for it and then complain